src/sql/workbench/parts/commandLine/electron-browser/commandLine.ts

```diff
@@ -26,8 +27,24 @@ import { INotificationService } from 'vs/platform/notification/common/notificati
 import { IWorkbenchContribution } from 'vs/workbench/common/contributions';
 import { IInstantiationService } from 'vs/platform/instantiation/common/instantiation';
 import { openNewQuery } from 'sql/workbench/parts/query/browser/queryActions';
+import { IURLService, IURLHandler } from 'vs/platform/url/common/url';
+import { parse } from 'vs/base/common/marshalling';
+import { getErrorMessage } from 'vs/base/common/errors';
+import { IDialogService } from 'vs/platform/dialogs/common/dialogs';
 
-export class CommandLineWorkbenchContribution implements IWorkbenchContribution {
+const connectAuthority = 'connect';
+
+interface SqlArgs {
+	_?: string[];
```

Comment:
What is this one for?

Reply:
It's needed by command line path, used to get all unnamed args (which are then treated as files). Using this interface let me avoid doing dummy inputs for all the commandline args, none of which this service uses